gemrb/core/GUI/GameControl.cpp

```diff
@@ -2079,6 +2075,15 @@ void GameControl::InitFormation(const Point &clickPoint)
 	SetCursor(core->Cursors[IE_CURSOR_USE]);
 }
 
+bool GameControl::TryDefaultTalk(Point p) const
+{
+	Actor *targetActor = core->GetGame()->GetCurrentArea()->GetActor(p,
+			target_types & ~(GA_NO_HIDDEN /*| GA_NO_ALLY | GA_NO_ENEMY*/ ));
+	if (targetActor)
+		targetActor->VerbalConstant(VB_SELECT, 4, DS_CIRCLE);
+	return true;
```

Comment:
The return value is never checked (or false), so you can make it void.

Reply:
i thought that in the future the return value could be of use somewhere, but ok

Reply:
Easier to add it then, if it's ever needed and we know what is actually needed. Perhaps it would even require an int to differentiate between more states.